cas/cas.h

```diff
@@ -135,6 +135,28 @@ struct m0_cas_kv_vec {
 	struct m0_cas_kv *cv_rec;
 } M0_XCA_SEQUENCE M0_XCA_DOMAIN(rpc);
 
+/**
+ * CAS record version and tombstone encoded in on-disk/on-wire format.
+ * Format:
+ *     MSB                             LSB
+ *     +----------------+----------------+
+ *     | 1 bit          | 63 bits        |
+ *     |<- tombstone -> | <- timestamp ->|
+ *     +----------------+----------------+
+ *
+ * A version comprises a physical timestamp and a tombstone flag.
+ * The timestamp is set on the client side whenever the corresponding
+ * record was supposed to be modified by PUT or DEL request.
+ * The tombstone flag is set when the corresponding record has been
```

Comment:
This flag is set by the CAS server internally, not the user, right?

Reply:
Yes. The catalogue store marks a record as dead if the user wants to delete the record.